exercises/rational-numbers/canonical-data.json

```diff
@@ -400,6 +400,28 @@
             "r": [0, 1]
           },
           "expected": 1.0
+        },
+        {
+          "uuid": "da137f1c-fc94-4035-9813-94f5152a1102",
+          "description": "Raise a negative real number to a rational number with an even numerator",
+          "property": "expreal",
+          "input": {
+            "x": -1,
+            "r": [2, 2]
+          },
+          "expected": 1.0
```

Comment:
I have some issues with this test
- the expected value should be `-1.0`
- the rational number is not reduced, and when it's reduced to `[1, 1]`, the numerator is odd